doc/usermanual/darkroom/modules/correction/denoise_profiled.xml

```diff
@@ -175,6 +220,37 @@
         within your image.
       </para>
     </sect5>
+    
+    <sect5>
+      <title>preserve shadows</title>
+      <para>
+	This option is available in <quote>wavelets</quote> and <quote>non local means</quote> modes.
+	It allows to denoise more agressively the shadows or the highlights. Lower the value to denoise
+	more the shadows than the highlights. Usually, as noise increases, you will need to lower this
+	value.
+      </para>
+    </sect5>
+    
+    <sect5>
+      <title>bias correction</title>
+      <para>
+	This option is available in <quote>wavelets</quote> and <quote>non local means</quote> modes.
+	It allows to correct the color cast that may appear in the shadows. Increase this value
+	if dark shadows appear too greenish, decrease it if they appear purple-ish.
+      </para>
+    </sect5>
+
+    <sect5>
+      <title>adjust autoset parameters</title>
+      <para>
+	This option is available in <quote>auto</quote> modes. In these modes, darktable tries to
+	infer denoising parameters from the camera profile. Yet, the automatically infered parameters
+	may be not optimal depending on your image. If your picture is very underexposed, you will have to
+	increase this parameter to get a proper denoising. This parameter should reflect the amplification
+	you add to your picture: if you add 1EV of exposure, the signal is multiplied by 2, so this
+	parameter should be set to 2.
```

Comment:
In these modes, darktable tries to derive denoising parameters from the camera profile. Depending on your image the automatically derived parameters may be not optimal. E.g. if your image is heavily underexposed and you lifted the exposure, you will have to increase this parameter to get a proper denoising. This parameter should reflect the amplification you add to your picture: if you add 1EV of exposure, the signal is multiplied by 2, so this parameter should be set to 2.